src/deploy-cromwell-on-azure/KubernetesManager.cs

```diff
@@ -224,7 +252,7 @@ private async Task<bool> WaitForWorkloadWithTimeout(IKubernetes client, string d
                 deployments = await client.AppsV1.ListNamespacedDeploymentAsync(configuration.AksCoANamespace, cancellationToken: cancellationToken);
                 deployment = deployments.Items.Where(x => x.Metadata.Name.Equals(deploymentName, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
                 
-                if (deployment.Status.ReadyReplicas == null || deployment.Status.ReadyReplicas < 1)
+                if (deployment == null || deployment.Status == null || deployment.Status.ReadyReplicas == null || deployment.Status.ReadyReplicas < 1)
```

Comment:
Can combine ?. with ?? to simplify this condition

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator